File: string/01.py
# EASY
# REMOVE VOWELS FROM A STRING

def removeVowels(word):
    word = word.lower()
    removed = ''
    for char in range(len(word)):
        choice = word[char]
        if (choice != 'a' and choice != 'e' and choice != 'i' and choice != 'o' and choice != 'u'):
            removed += choice
    return removed


result = removeVowels('Aeroplane')
print(result)

File: string/01.js
// EASY
// REMOVE VOWELS FROM A STRING

const removeVowels = function (word) {
    let removed = '';
    for (char of word.toLowerCase()) {
        if (!(char == 'a' || char == 'e' || char == 'i' || char == 'o' || char == 'u')) {
            removed += char;
        }
    }
    return removed;    
}

const result = removeVowels('Aeroplane');
console.log(result);
